src/common/setup.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+ 
+from distutils.core import setup
+
+setup(
+    name='common',
+    packages=['common']
+)
```

Comment:
setup.py at root is mandatory to run 'pip install -e' and then import between different packages